### PR TITLE
Dg 25639

### PR DESCRIPTION
**Pull request by glitchinfinity**

### CODE REVIEW

File: spec/customer_standard_domain_spec.rb [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+
+    it 'calls the customer standard domains enable endpoint with the site_nonprod template name' do
+      stub_request(:any, /.*#{@mock_endpoint}.*#{path}/)
+        .with(headers: @mock_headers)
+        .to_return { |request| { body: { uri: request.uri, body: request.body, method: request.method }.to_json } }
+      payload = ['template1-[sitename]-[environment].example.com']
+      res = @conn.customer_standard_domain.enable_customer_standard_domain_template('site_nonprod', payload)
+      uri = URI res['uri']
+      expect(uri.path).to eq path
+      expect(res['method']).to eq 'put'
```
Comment (alanevans) [inline spec/customer_standard_domain_spec.rb:35]: Can we or is it worth asserting the payload as well?

Comment (alanevans) [reply spec/customer_standard_domain_spec.rb:35]: Assuming that might come through in res['body']?

Comment (glitchinfinity) [reply spec/customer_standard_domain_spec.rb:35]: Fixed :D

**Comment by glitchinfinity** [comment]
Closing this as we are going to create a new one with the renamed API name.

**Pull request closed without merging**